src/codegen/c.zig

```diff
@@ -386,52 +386,97 @@ pub const DeclGen = struct {
         try dg.renderDeclName(decl, writer);
     }
 
-    fn renderInt128(
+    fn renderBigIntConst(
```

Comment:
What is the reason to rename it like it would accept any integer sizes and then do `assert(bits <= 128);` in the fn body?

Reply:
The reason I did those changes (`renderInt128`/`renderBigIntConst` refactoring/renaming) is not obvious anymore, I'll remove unneeded changes.